React/Base/RCTTouchHandler.m

```diff
@@ -150,6 +150,13 @@ - (void)_updateReactTouchAtIndex:(NSInteger)touchIndex
   reactTouch[@"locationX"] = @(touchViewLocation.x);
   reactTouch[@"locationY"] = @(touchViewLocation.y);
   reactTouch[@"timestamp"] =  @(nativeTouch.timestamp * 1000); // in ms, for JS
+  
+  if ([nativeTouch.view respondsToSelector:@selector(traitCollection)] &&
+      [nativeTouch.view.traitCollection respondsToSelector:@selector(forceTouchCapability)] {
```

Comment:
@2upmedia yes. As stated in the comments I am waiting on a final API decision to update the PR